pom.xml

```diff
@@ -41,6 +41,7 @@
     <properties>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.release>8</maven.compiler.release>
```

Comment:
This causes [MCOMPILER-339](https://issues.apache.org/jira/browse/MCOMPILER-339) when building using Java 8 (see [this comment](https://github.com/saveourtool/diktat/pull/1534#issuecomment-1295254240)).